File: Programmers/Lv.0/한 번만 등장한 문자/120896.java
import java.util.Arrays;

class Solution {
    public String solution(String s) {
        String answer = "";
        
        String[] arr = s.split("");
        
        Arrays.sort(arr);
        
        for (int i=0; i<arr.length-1; i++) {
            String str = arr[i];
            for (int j=i+1; j<arr.length; j++) {
                if (str.equals(arr[j])) {
                    arr[i] = "";
                    arr[j] = "";
                }
            }
        }
        
        answer = String.join("", arr);
        
        return answer;
    }
}

File: Programmers/Lv.0/한 번만 등장한 문자/120896.js
function solution(s) {
    let answer = '';
    
    const arr = [...s].sort();
    for (let i=0; i<arr.length-1; i++) {
        const str = arr[i];
        for (let j=i+1; j<arr.length; j++) {
            if (str === arr[j]) {
                arr[i] = '';
                arr[j] = '';
            }
        }
    }
    
    answer = arr.join('');
    
    return answer;
}
